docs-source/content/security/openshift.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "OpenShift"
+date: 2019-10-04T08:08:08-05:00
+weight: 7
+description: "OpenShift information for the WebLogic operator"
```

Comment:
The official product name is the Oracle WebLogic Kubernetes Operator. I think you can get away with calling it the WebLogic Kubernetes Operator or more easily, just refer to it as the operator.

Reply:
This is what is in all the existing .md files in the security section, do I change it for all of them or leave alone for now?

Reply:
Done